src/main/java/com/dtolabs/rundeck/plugin/overthere/OTWinRMNodeExecutor.java

```diff
@@ -229,6 +240,8 @@ public NodeExecutorResult executeCommand(final ExecutionContext context, final S
             return NodeExecutorResultImpl.createFailure(StepFailureReason.ConfigurationFailure, e.getMessage(), node);
         }
 
+        String cmdType = options.get(WINRM_CMD_TYPE);
```

Comment:
Move this 2-lines block where the previous log was, so to have the same behavior as previously (with the patch as is, if the try-block excepts, the log is not written, while it was before)